nifi-nar-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/src/main/java/org/apache/nifi/dbcp/DBCPConnectionPool.java

```diff
@@ -164,6 +161,71 @@ public ValidationResult validate(final String subject, final String input, final
         .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
         .build();
 
+    public static final PropertyDescriptor MIN_IDLE = new PropertyDescriptor.Builder()
+            .name("Minimum Idle Connections")
+            .description("The minimum number of connections that can remain idle in the pool, without extra ones being " +
+                    "created, or zero to create none.")
+            .defaultValue("0")
+            .required(true)
+            .addValidator(StandardValidators.INTEGER_VALIDATOR)
+            .sensitive(false)
+            .build();
+
+    public static final PropertyDescriptor MAX_IDLE = new PropertyDescriptor.Builder()
+            .name("Max Idle Connections")
+            .description("The maximum number of connections that can remain idle in the pool, without extra ones being " +
+                    "released, or negative for no limit.")
+            .defaultValue("8")
```

Comment:
Setting this to `8` feels so weird, even though it is legitimately the default value in the DBCP library.

Reply:
There are some defaults I could reference, e.g. `org.apache.commons.pool2.impl.BaseObjectPoolConfig#DEFAULT_MIN_EVICTABLE_IDLE_TIME_MILLIS` and `org.apache.commons.pool2.impl.GenericObjectPoolConfig#DEFAULT_MAX_IDLE`. Should I instead reach into DBCP to get those?

Reply:
@mattyb149 If you have a second, I'd appreciate your thoughts on this.

Reply:
IMO we should keep the default value as the one used when you don't explicitly set it, I guess that's 8?

Reply:
@mattyb149 Yes, it's 8. Should I reach into the impl to refer to the default constants?